templates/index.html

```diff
@@ -102,16 +102,13 @@ <h5 class="accordion-header">
           job openings.") }}
           <ul class="list-group list-group-flush">
             {% for company in companies %}
-            <li class="list-group-item list-group-item-action">
-              <a href="{{ company.link }}" target="_blank">{{ company.name }}</a>
-              <ul class="list-group">
+            <div class="list-group-item">
```

Comment:
```suggestion
            <li class="list-group-item">
```
So that it is semantically correct, using `<div>` might interfere with screen readers in theory.

Reply:
Right, fixed...